src/Widget/WidgetInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Widget;
+
+use Symfony\Component\HttpFoundation\Response;
+
+interface WidgetInterface
+{
+    public function getName(): string;
+
+    public function getType(): string;
+
+    public function getTarget(): string;
+
+    public function getPriority(): int;
+
+    public function getTemplate(): string;
+
+    public function setResponse(?Response $response = null): self;
+
+    public function getResponse(): ?Response;
+
+    public function getZone(): string;
+
+    public function getSlug(): string;
+
+    public function __invoke(?string $template = null): string;
```

Comment:
what is a template?

Reply:
A template is either a string that can be used by a widget in it's own `__invoke()` method as either a plain string or a simple template to be used in `sprintf()`, or it can be the filename of a twig template, like `@bolt/widgets/weather.twig`

Reply:
Can you add this as a comment there?